File: table.lua
-- take table info from user

io.write("Enter table: ")
local table = io.read()

io.write("Enter table start: ")
local table_start = io.read()

io.write("Enter table end: ")
local table_end = io.read()

-- generate table based on info

for table_term=table_start, table_end do
    print(string.format("%d x %d = %d", table, table_term, table * table_term))
end

File: table.py
# take user input for table info

table = int(input("Enter table: "))
table_start = int(input("Enter table start: "))
table_end = int(input("Enter table end: "))

# generate table based on info

for table_term in range(table_start, table_end+1):
    print(f"{table} x {table_term} = {table * table_term}")
